metaserver/src/main/scala/scala/meta/languageserver/refactoring/OrganizeImports.scala

```diff
@@ -25,17 +26,24 @@ object OrganizeImports extends LazyLogging {
   def removeUnused(
       arguments: Option[Seq[JsValue]],
       index: SymbolIndex
-  ): ApplyWorkspaceEditParams = {
+  ): Task[Either[Response.Error, ApplyWorkspaceEditParams]] = {
     val result = for {
       as <- arguments
       argument <- as.headOption
       textDocument <- Json.fromJson[TextDocumentIdentifier](argument).asOpt
     } yield removeUnused(Uri(textDocument), index)
-    result.getOrElse(
-      throw InvalidParamsResponseError(
-        s"Unable to parse TextDocumentIdentifier from $arguments"
-      )
-    )
+    Task {
```

Comment:
Why does this have to be a task if the result is returned immediately? Also, if this has to be a task, then why not wrap the computation of `result`?

Reply:
I agree, I would either wrap the whole thing in `Task` or use `Task.now` if this is synchronous.